Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -405,8 +405,65 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
         },
     });
 
+    getEditViewParametersTest = new Y.Test.Case({
+        name: "eZ Location View View Service getViewParameters tests",
+
+        setUp: function () {
+            this.app = new Y.Mock();
```

Comment:
why a mock for app ? it seems to not be configured anywhere, so you can probably use an empty object instead.